FEATURES.md

```diff
@@ -27,4 +27,17 @@ Use the filter option to find the desired chart quicker from the list of charts.
 ![Repository6](images/screenshot_repository6.png)
 
 If you want to install a particular chart, simply hover the pointer over the chart name and an 'Install' button will appear, as shown in the figure.
-![Repository7](images/screenshot_repository7.png)
\ No newline at end of file
+![Repository7](images/screenshot_repository7.png)
+
+# Installed Releases list
+This tells us which chart version we are using.
```

Comment:
In terminology of Helm, that list is list of _releases_ installed. Try to describe this "list of releases" idea a bit better. This includes describing the filtering by namespace and filtering by text.